openmc/mesh.py

```diff
@@ -165,7 +165,7 @@ def _grids(self):
         pass
 
     @property
-    def vertices(self):
+    def vertices(self):  # TODO needs changing
```

Comment:
@pshriwise do we need to adapt this or can it remain in cylindrical/spherical coordinates?

Reply:
We'll want to make sure we translate all of the vertices at the end using the origin if it is set. Same for `centroids.

Reply:
Hm. I don't see where the conversion from cylindrical to cartesian is done for the vertices.
It doesn't seem to be exported to xml so it's unused in the C++ part am I right?

Currently, it doesn't seem that vertices (or centroids) of a CylindricalMesh are converted to cartesian (at least not in mesh.py).

Reply:
According to @eepeterson we don't need to adapt this